core/arch/arm/kernel/link.mk

```diff
@@ -28,7 +28,8 @@ link-script-cppflags := -DASM=1 \
 entries-unpaged += thread_init_vbar
 entries-unpaged += sm_init
 entries-unpaged += core_init_mmu_regs
-entries-unpaged += stack_tmp_top
+entries-unpaged += stack_tmp
```

Comment:
OK, will update.
`KEEP_PAGER(x)` and `entries-unpaged += x` are not the same, what is the rule? It looks like we're adding functions to `entries-unpaged` but data are for `KEEP_PAGER`?